spec/requests/comments_create_spec.rb

```diff
@@ -89,21 +89,42 @@ def comment_params(**kwargs)
   end
 
   context "when user is commenting on a comment by an author who has blocked the user" do
-    let(:third_party_article) { create(:article, user_id: create(:user).id) }
-
-    it "raises a ModerationUnauthorizedError to prevent the comment from saving" do
+    before do
       create(:user_block, blocker: blocker, blocked: user, config: "default")
       # Manually manage the blocked_by_count attribute that counter_culture manages in prod
       user.update(blocked_by_count: 1)
```

Comment:
Maybe `update_column` to avoid callbacks/validations? (if applicable)

Reply:
I think that's a good and very inexpensive improvement! Thanks!